Cargo.toml

```diff
@@ -9,17 +9,18 @@ edition = "2021"
 async-tungstenite = "0.23.0"
 cfg-if = "1.0.0"
 clap = { version = "4.4.6", features = ["derive", "env"] }
-crossbeam-channel = "0.5.8"
 eyre = "0.6.8"
-futures-util = { version = "0.3.28", default-features = false, features = ["sink"]}
 paste = "1.0.14"
 rosu-pp = "0.9.5"
 serde = { version = "1.0.189", features = ["derive"] }
 serde_json = "1.0.107"
 serde_repr = "0.1.16"
 smol = "1.3.0"
-#tracy-client = { version = "0.16.4", default-features = false, features = ["enable"] }
+futures-util = { version = "0.3.28", default-features = false, features = ["sink"] }
 tracy-client = { version = "0.16.4", default-features = false }
+async-dup = "1.2.2"
+hyper = { version = "0.14.27", features = ["server", "http1"] }
+async-compat = "0.2.3"
```

Comment:
`hyper` uses `tokio` as dependency so it will now be in the dependency tree while `smol` is still being used. Although not strictly necessary, at this point it might be better to throw out `smol` and replace it with tokio completely. This should also render `async-compat` redundant futures would only be run by tokio anyway.

Reply:
`async-compat` is gone. `hyper` now uses tokio only for sync primitives so i think is fine.
Also i think switching to `tokio`  is too much for this project and don't really worth it (especially since `hyper` 1.0 introduced `hyper::rt::{Read, Write}` traits)